packages/react-core/src/demos/examples/Tabs/ModalTabs.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import DashboardWrapper from '../DashboardWrapper';
+import {
+  PageSection,
+  TextContent,
+  Text,
+  Gallery,
+  Card,
+  CardBody,
+  CardTitle,
+  PageSectionVariants,
+  Modal,
+  ModalVariant,
+  Tab,
+  Tabs,
+  TabTitleText,
+  List,
+  ListItem,
+  Grid,
+  GridItem,
+  TabContent
+} from '@patternfly/react-core';
+
+interface Product {
+  id: number;
+  name: string;
+  description: string;
+}
+
+const products: Product[] = [
+  {
+    id: 0,
+    name: 'Patternfly',
```

Comment:
match the case in the description

```suggestion
    name: 'PatternFly',
```